includes/class-wc-payments-features.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Class WC_Payments_Features
+ *
+ * @package WooCommerce\Payments
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * WC Payments Features class
+ */
+class WC_Payments_Features {
```

Comment:
I like this new class.

Reply:
I wasn't sure whether to introduce it or not. On one hand I'd like it to keep the feature testing scoped in here, so that other classes can reference it. But on the other hand I'm not sure if we're exposing too much to external consumers. Either way, somebody that is more tech-savvy could trick the system and test feature flags 🤷